README.adoc

```diff
@@ -1,14 +1,8 @@
 = FSL Community BSP for Christ Electronic Systems Devices
 
-Yocto System requirements can be found https://www.yoctoproject.org/docs/current/ref-manual/ref-manual.html#ref-manual-system-requirements[here] and required packages for the build https://www.yoctoproject.org/docs/current/ref-manual/ref-manual.html#required-packages-for-the-build-host[here]
+Yocto System requirements can be found https://www.yoctoproject.org/docs/current/ref-manual/ref-manual.html#ref-manual-system-requirements[here].
 
-For an Ubuntu Build Host you need the following packages:
-
-[source,console]
-$: sudo apt-get install gawk wget git-core diffstat unzip texinfo gcc-multilib g++-multilib \
-build-essential chrpath socat cpio python python3 python3-pip python3-pexpect \
-xz-utils debianutils iputils-ping python3-git python3-jinja2 libegl1-mesa libsdl1.2-dev \
-pylint3 xterm
```

Comment:
Our experience showed that people often don't follow these links.
That's the reason we added this list. Please re-add the list, thanks.